config/locales/ar.yml

```diff
@@ -96,10 +97,12 @@ ar:
       scotland_slug:
     next_holiday_in_is:
     past_bank_holidays:
+    print_link:
     substitute_day:
     today:
     united-kingdom_slug:
     upcoming_bank_holidays:
+    visit: 'تفضّل بزيارة:'
```

Comment:
It's possible this command was a bit too broad - it's brought in this common.visit, which isn't used in this partial. Perhaps run the command again twice, once for common.print_link, once for common.email_and_print_link, so it doesn't bring in other things?